nile-backend/src/main/resources/application-dev.properties

```diff
@@ -1,5 +1,5 @@
 spring.datasource.driverClassName=org.postgresql.Driver
-spring.datasource.url=jdbc:postgresql://localhost:5432/Nile
+spring.datasource.url=jdbc:postgresql://localhost:5432/NewNile
```

Comment:
I think We should keep it the way that it had been set. You can always remove a file from staging in a commit with `git reset [filename]`.